libvirt/tests/src/daemon/conf_file/sysconfig_libvirt_guests/libvirt_guests.py

```
@@ -242,11 +254,28 @@ def cleanup_nfs_backend_guest(vmxml_backup):
 
         nfs_server = libvirt.setup_or_cleanup_nfs(False)
 
+    def transfer_to_transient(per_guest_name):
+        """
+        Transfer one persistent vm to transient guest.
+
+        :param  per_guest_name: persistent guest name.
+        """
+        virsh.start(per_guest_name)
```

`ignore_status=False` could fail fast if vm fails to start